### PR TITLE
Mihai development

### PR DESCRIPTION
**Pull request by PredaMihaiDragos**

### CODE REVIEW

**Review by martemyev — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -9,9 +9,16 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
+
       <p>Hello, my name is Mihai!</p>
+      <p>I am from Romania and I am a student at the University of Bucharest in Computer Science</p>
+      <p>I've just ended my first year and I think it was awesome!</p>
```
Comment (martemyev) [inline portfolio/src/main/webapp/index.html]: 'I just finished' sounds better :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -16,6 +16,26 @@ <h1>My Portfolio</h1>
...
+            <b>QuizGame</b> 
+            <p>- A project for the OOP class, made with C++20 using latest features and a looot of OOP.</p>
+            <p>- Three types of questions (MultipleChoice, Word and Number) you have to answer.</p>
+            <p>- You can also add questions to the game.</p>
+            <p>- Can be found <a href="https://github.com/PredaMihaiDragos/Facultate/tree/master/Anul%201/POO/QuizGame">here</a></p>
+          </li>
+          <li>
+            <b>Website for blood analysis lab</b>
+            <p>- A project for both the Web class and my mother who has a blood analysis lab. </p>
+            <p>- Frontend: we were not allowed any APIs or to use any template, so it's plain HTML / CSS (with flexbox and grid) / JS written from 0. </p>
```
Comment (martemyev) [inline portfolio/src/main/webapp/index.html]: 'we were not allowed to use any APIs or templates'

Comment (martemyev) [inline portfolio/src/main/webapp/index.html]: 'from scratch' :)

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -16,6 +16,26 @@ <h1>My Portfolio</h1>
...
+            <p>- Backend: node.js </p>
+            <p>- Looks fairly good, has things like modal boxes, regex field validation, local storage, lots of AJAX and JS events, pages generated from templates etc</p>
+            <p>- Completely responsive on mostly any resolution!</p>
```
Comment (martemyev) [inline portfolio/src/main/webapp/index.html]: Not sure how responsiveness is related to the resolution. Can you rephrase it?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -16,6 +16,26 @@ <h1>My Portfolio</h1>
...
+            <p>- Can be found <a href="https://github.com/PredaMihaiDragos/Facultate/tree/master/Anul%201/TW/Proiect">here</a></p>
```
Comment (martemyev) [inline portfolio/src/main/webapp/index.html]: As this project is in Romanian, I'd suggest to mention it somewhere. Maybe just in parens next to the link: Can be found here (note, it is in Romanian)

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -2,11 +2,20 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Mihai Preda's portfolio</title>
     <link rel="stylesheet" href="style.css">
+
     <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/fe29c7813b.js" crossorigin="anonymous"></script>
```
Comment (martemyev) [inline portfolio/src/main/webapp/index.html:10]: What's this for?

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -69,18 +69,6 @@ <h2>Accounts</h2>
             <p>My Github: <a href="https://github.com/PredaMihaiDragos">Here</a></p>
       </section>
 
-      <section>
-          <h2>Images</h2>
-            <img class="photo-of-me" src="images/photo_of_me.jpeg" alt="Photo of me">
```
Comment (martemyev) [inline portfolio/src/main/webapp/index.html]: You'll also need to delete the jpeg file

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -26,3 +26,33 @@ function addRandomFact() {
...
+}
+
+
+/**
+ * Function to call when page loaded to init javascript stuff
+ */
+function init() {
+    // Simulate scroll event to position elements right
+    OnWindowScrolled();
+}
```
Comment (martemyev) [inline portfolio/src/main/webapp/script.js]: No new line at the end of the file. You may want to add it.

File: portfolio/src/main/webapp/script.js [hunk 8]
```diff
@@ -26,3 +26,33 @@ function addRandomFact() {
...
+    }
+    else {
+        navBar.classList.remove("fixedNav");
+        content.style.paddingTop = 0;
+    }
+}
+
+
+/**
+ * Function to call when page loaded to init javascript stuff
```
Comment (martemyev) [inline portfolio/src/main/webapp/script.js]: Please rephrase the comment, it is a bit hard to read.

File: portfolio/src/main/webapp/script.js [hunk 9]
```diff
@@ -26,3 +26,33 @@ function addRandomFact() {
...
+ */
+function init() {
```
Comment (martemyev) [inline portfolio/src/main/webapp/script.js:57]: Is it currenty called anywhere?

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (PredaMihaiDragos) [reply portfolio/src/main/webapp/index.html:10]: That is for the icons in the header (mail, phone, facebook, linkedin and github)

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (PredaMihaiDragos) [reply portfolio/src/main/webapp/index.html]: Not sure if it's a correct translation, but "responsive" is the word we used at the university for "web pages render well on a variety of devices and window or screen sizes" (quote from [wiki](https://en.wikipedia.org/wiki/Responsive_web_design)), so making media queries for different resolutions.
Does this sound better?
`
Completely responsive, works well on mostly any screen size!
`

File: portfolio/src/main/webapp/script.js [hunk 10]
```diff
@@ -13,16 +13,16 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['My last name is Preda!', 'I\'m from Romania!', 'I like programming！', 'My hobby is traveling!', 'When I was a kid, I wanted to be a truck driver!'];
```
Comment (saiavin) [inline portfolio/src/main/webapp/script.js]: Is the space after "programming!" required?

**Review by saiavin — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/script.js (hunk 10 above)
Comment (PredaMihaiDragos) [reply portfolio/src/main/webapp/script.js]: Well, there is no actual space after programming (try to select it), I don't know why it looks like this.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (martemyev) [reply portfolio/src/main/webapp/index.html]: Got it, thanks. 'Screen size' sounds more clear in this case.

File: portfolio/src/main/webapp/script.js (hunk 10 above)
Comment (martemyev) [reply portfolio/src/main/webapp/script.js]: It looks like a non-ASCII exclamation mark. Hence it is shown like this. Please delete it and replace with an ASCII one.
